.gitmodules

```diff
@@ -80,7 +80,7 @@
 	url = https://github.com/sonic-net/sonic-mgmt-framework
 [submodule "src/sonic-telemetry"]
 	path = src/sonic-telemetry
-	url = https://github.com/sonic-net/sonic-telemetry
+	url = https://github.com/sonic-net/sonic-gnmi
```

Comment:
This change doesn't look right.

Reply:
sonic-telemetry is archived. I pushed sonic-telemetry's code to sonic-gnmi's 202205 branch.